docs/Geographic_redundancy.md

```diff
@@ -15,7 +15,7 @@ Each site has its own, separate, etcd cluster. This means that Clearwater's [aut
 Vellum has 3 databases, which support Geographic Redundancy differently:
 
 * The Homestead-Prov, Homer and Memento databases are backed by Cassandra, which is aware of local and remote peers, so these are a single cluster split across the two geographic regions.
-* Chronos is aware of local peers and the remote cluster, and handles replicating timers across the two sites itself.
+* Chronos is aware of local peers and the remote cluster, and handles replicating timers across the two sites itself if configured to be GR. (By default it is configured not to be GR - details [here](https://github.com/Metaswitch/chronos/blob/dev/doc/gr.md).)
```

Comment:
Mm. This is a bit clunky. How about 'There is one Chronos cluster per geographic region. By default, Chronos does not replicate timers to remote clusters, as timers are not sufficiently critical to require that level of redundancy. If desired, Chronos can be made aware of remote clusters and will handle replicating timers to the remote sites itself (details [here]).'

Reply:
I do like that better. I've made this change.